src/pfe/file-watcher/server/src/controllers/projectStatusController.ts

```diff
@@ -90,6 +93,8 @@ class ProjectState {
             this.buildImageLastBuild = buildImageLastBuild;
         if (detailedAppStatus)
             this.detailedAppStatus = detailedAppStatus;
+        if (notificationID)
```

Comment:
Think this can be updated to
```
if (!(notificationID in this.detailedAppStatus))
            this.detailedAppStatus.notificationID = "";
```
then we don't need to set `notificationID` to `""` in other places in the code as well. Just need to set it when it should be something other than default?